src/keyexpr.rs

```diff
@@ -24,7 +24,7 @@ use zenoh::{
 pub use crate::opaque_types::{z_loaned_keyexpr_t, z_owned_keyexpr_t, z_view_keyexpr_t};
 use crate::{
     errors,
-    errors::{z_error_t, Z_OK},
+    errors::{z_result_t, Z_OK},
```

Comment:
do we also want to change errors module name, or is it ok to leave it as is ?

Reply:
I've changed it to `result`.